lxd/metrics/metrics.go

```diff
@@ -92,6 +92,7 @@ func (m *MetricSet) String() string {
 	gaugeMetrics := []MetricType{
 		ProcsTotal,
 		CPUs,
+		APIOngoingRequests,
```

Comment:
any reason for this ordering?

Reply:
No reason in particular, I will move it to the end of the list.